ci/tests/gui/basic_test.py

```diff
@@ -306,15 +310,19 @@ def create_from_template_test():
     hpv.poweroff(machine_name)
     bt.set_as_template(machine_name, allowed=True)
 
-    template = Vmachine.get_template(machine_name, vpool_name)
+    time.sleep(60)
 
-    browser_object = bt = Vmachine()
-    bt.login()
+    try:
+        template = Vmachine.get_template(machine_name, vpool_name)
 
-    bt.create_from_template(template.name, name)
-    bt.check_machine_is_present(name)
+        browser_object = bt = Vmachine()
+        bt.login()
 
-    hpv.delete(name)
+        bt.create_from_template(template.name, name)
+        bt.check_machine_is_present(name)
+        hpv.delete(name)
+    except Exception as ex:
```

Comment:
2 questions:
1) Why put everything in try - except and just print the error if any?  Test will succeed even create from template failed
2) Why sleep 60 seconds? If the template is not available right after set_as_template completed, something else will be wrong and watiing a minute won't fix it either